docker_test.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"testing"
+
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	"github.com/jfrog/jfrog-cli-go/artifactory/commands/generic"
+	"github.com/jfrog/jfrog-cli-go/artifactory/spec"
+	"github.com/jfrog/jfrog-cli-go/inttestutils"
+	"github.com/jfrog/jfrog-cli-go/utils/cliutils"
+	"github.com/jfrog/jfrog-cli-go/utils/tests"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const DockerTestImage string = "jfrog_cli_test_image"
+
+// Image get parent image id command
+type buildDockerImage struct {
+	dockerFilePath string
+	dockerTag      string
+}
+
+func (image *buildDockerImage) GetCmd() *exec.Cmd {
+	var cmd []string
+	cmd = append(cmd, "docker")
+	cmd = append(cmd, "build")
+	cmd = append(cmd, image.dockerFilePath)
+	cmd = append(cmd, "--tag", image.dockerTag)
+	return exec.Command(cmd[0], cmd[1:]...)
+}
+
+func (image *buildDockerImage) GetEnv() map[string]string {
+	return map[string]string{}
+}
+
+func (image *buildDockerImage) GetStdWriter() io.WriteCloser {
+	return nil
+}
+
+func (image *buildDockerImage) GetErrWriter() io.WriteCloser {
+	return nil
+}
+
+func InitDockerTests() {
+	if !*tests.TestDocker {
+		return
+	}
+	os.Setenv(cliutils.ReportUsage, "false")
+	os.Setenv(cliutils.OfferConfig, "false")
```

Comment:
Setting the above two environment variables can be placed in the beginning of the *setupIntegrationTests()* function. That will allow us to remove the *InitDockerTests()* function, as well as the other init functions, which include only these two env vars set.